### DISCUSSION TITLE
[BUG] - Autocomplete: selectionMode is not working

### DISCUSSION THREAD

**Discussion by DeshChow** [heroui-inc/heroui#4746] [Feature requests]
### NextUI Version

2.1.10

### Describe the bug

here, in the code example , selectionMode is used, but it is not working 

### Steps to Reproduce the Bug or Issue

Use this codeblock of AutoComplete and try to use selectionMode on it

### Expected behavior

selectionMode should work

### Operating System Version

macOs

### Browser

Chrome

**Comment by black197** [comment]
I tried `@nextui-org/react@2.1.10` in Next.js but it says there's no Autocomplete exported from `@nextui-org/react`. And I found that `Autocomplete` became available in v2.2.0(see #1777). There's also another PR(#1857) about selectionMode but it's not merged yet.

**Comment by TateWalker** [comment]
There's an open Feature Request (#2016) for this
